.circleci/config.yml

```diff
@@ -477,20 +546,31 @@ jobs:
     environment:
       JVM_OPTS: -Xmx1024m
     steps:
+      - skip_job_if_required
       - run:
-          name: check environment variabe 
+          name: reset OVERRIDE_TEST environment variabe 
           command: |
-              if [ "${SKIP_INTEGRATIONTEST}" == "TRUE" ]; then
-                echo "project enviroment variabe SKIP_INTEGRATIONTEST should not be TRUE at the moment, this will skip all integration tests"
-                exit 1
-              fi           
+            parameters='{"name":"OVERRIDE_TEST","value":"FALSE"}'
```

Comment:
use first 7 chars of commit hash in variable name

Reply:
done